e2e/integration-test/e2e_metric_test.go

```diff
@@ -260,27 +271,39 @@ var _ = Describe("Metrics check should pass", Ordered, func() {
 		Expect(err).NotTo(HaveOccurred())
 	})
 
-	_ = DescribeTable("Check node level metrics for details",
+	_ = DescribeTable("Check node level metrics for details, non zero value metric should be found",
 		func(metricName string) {
-			checkMetricValues(keplerMetric, metricName)
+			noneZeroMetricValues(keplerMetric, metricName)
+		},
+		Entry(nil, "kepler_exporter_build_info"),       // only one
+		Entry(nil, "kepler_node_core_joules_total"),    // node level
```

Comment:
The test is failing for `kepler_node_core_joules_total`. This metric is expected to be 0 right?

Reply:
only node platform and uncore are expected to be 0

Reply:
my env shows non zero, is it because something in CI env?

Reply:
not sure what happened to the CI, it is always 0

Reply:
fixed